specs/core/1_shard-data-chains.md

```diff
@@ -106,7 +106,7 @@ class ShardBlock(Container):
     shard: Shard
     beacon_chain_root: Hash
     parent_root: Hash
-    data: ShardBlockBody
+    body: ShardBlockBody
```

Comment:
We should discuss just removing the one-item `ShardBlockBody` struct and making it directly be a `Bytes` during the call :smile:

Reply:
probably reasonable..